ts/Extensions/Annotations/Popup.ts

```diff
@@ -165,6 +166,41 @@ var indexFilter = /\d/g,
     LI = 'li',
     H3 = 'h3';
 
+// Lang definitions for indicators parameters in Stock Tools popup, #15170
+setOptions({
```

Comment:
Rather question than request.
Is there any difference in performance or anything else between `setOptions` and `merge` like below?
```
merge(defaultOptions, {
    lang: {
       ...
    }
})
```

Reply:
Yes. `setOptions` changes (extends) the first argument when merge only returns the whole new object (without assigning or changing the first one). So you would have to either do something like this:
```js
defaultOptions = merge(defaultOptions, {
    lang: {
       ...
    }
})
```
or
```js
merge(true, defaultOptions, {
    lang: {
       ...
    }
})
```

https://api.highcharts.com/class-reference/Highcharts#.merge%3CT%3E

Reply:
Ok, `setOptions` looks correctly in this case! Thanks ;)